get reference to fastlane folder the more correct way

taquitos opened the pull request:
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Description
Update to get the fastlane folder the more correct way (that will even find it if the folder starts with a .)

Commit:
get reference to fastlane folder the more correct way

KrauseFx commented on fastlane_core/lib/fastlane_core/helper.rb:
```diff
@@ -240,6 +240,10 @@ def self.fastlane_enabled?
       @enabled ||= !FastlaneCore::FastlaneFolder.path.nil?
     end
 
+    def self.fastlane_enabled_folder_path
```
A comment here would be great to describe what the method does

KrauseFx reviewed (approved):
Wohooo, all the clean ups :recy

Commit:
Sure rubocop. Whatever.
Fixes:
deliver/lib/deliver/commands_generator.rb:38:5: W: Lint/UnneededDisable: Unnecessary disabling of Metrics/PerceivedComplexity.

taquitos committed:
Added comment

Describes fastlane_enabled_folder_path

ohayon reviewed (approved):
💯

taquitos committed:
Cleaned up rubocop's mess

Added new line

KrauseFx reviewed (approved):
:fidget_spinner:

taquitos commented:
@KrauseFx omg when are we getting fastlane fidget spinners?
